web/app.js

```diff
@@ -560,6 +560,18 @@ const PDFViewerApplication = {
 
     if (appConfig.annotationEditorParams) {
       if (annotationEditorMode !== AnnotationEditorType.DISABLE) {
+        if (AppOptions.get("enableStampEditor") === true) {
+          const editorStampButton = appConfig.toolbar?.editorStampButton;
+          if (
+            editorStampButton &&
+            AppOptions.get("enableStampEditor") &&
+            AppOptions.get("isOffscreenCanvasSupported") &&
+            FeatureTest.isOffscreenCanvasSupported
+          ) {
+            editorStampButton.hidden = false;
+          }
+        }
+
```

Comment:
There's some duplication here, so we should be able to remove the outer most `if`-check now.
```suggestion
        const editorStampButton = appConfig.toolbar?.editorStampButton;
        if (
          editorStampButton &&
          AppOptions.get("enableStampEditor") &&
          AppOptions.get("isOffscreenCanvasSupported") &&
          FeatureTest.isOffscreenCanvasSupported
        ) {
          editorStampButton.hidden = false;
        }
 ```